src/app/routes/utils/overrideRendererOnTest/index.js

```diff
@@ -0,0 +1,18 @@
+import Url from 'url-parse';
+import isLive from '#lib/utilities/isLive';
+
+export const addOverrideQuery = (path) => {
+  const url = new Url(path);
+  const { query } = url;
+  const searchParams = new URLSearchParams(query);
```

Comment:
is this safe to use? https://caniuse.com/#search=URLSearchParams

Reply:
It's not and I've replaced it